### DISCUSSION TITLE
Puma never utilizes 100% CPU

### DISCUSSION THREAD

**Discussion by jeffsaremi** [puma/puma#3206] [Q&A] [answered]
**Describe the bug**
We set puma to use the same number of workers as the number of CPU cores on our machines
We load the instance with requests to the max; the CPU reaches 90% and plateaus there

**Desktop (please complete the following information):**
 - OS: linux
 - Puma Version [e.g. 4.1.1]

**Comment by dentarg** [comment, marked as answer]
90% sounds pretty good to me!

To be honest, I don't know what you expected from opening an issue with that tiny amount of information. You are running an End-of-Life version of Puma too. @jeffsaremi, you need to step up your game 😉

**Reply by jeffsaremi** [reply]
yes. thank you! I forgot this option was also available
